cmd/cni-metrics-helper/metrics/metrics.go

```diff
@@ -426,4 +453,7 @@ func Handler(ctx context.Context, t metricsTarget) {
 
 	cw := t.getCWMetricsPublisher()
 	produceCloudWatchMetrics(t, families, interestingMetrics, cw)
```

Comment:
Shouldn't this be blocked by `submitCloudWatch()`?

Reply:
This has been there since day one and there are bunch of places we are checking https://github.com/jayanthvn/amazon-vpc-cni-k8s/blob/master/cmd/cni-metrics-helper/metrics/metrics.go#L322 inside the function and hence it has not been the issue..I will move it out and should be safe..